synapse/module_api/__init__.py

```diff
@@ -700,6 +715,38 @@ def read_templates(
             (td for td in (self.custom_template_dir, custom_template_directory) if td),
         )
 
+    def ts(self) -> int:
+        """Return a timestamp for the current time, in milliseconds since the epoch"""
+        return self._hs.get_clock().time_msec()
+
+    def is_mine(self, domain_specific_string: DomainSpecificString) -> bool:
+        """Checks whether an ID comes from this homeserver."""
+        return self._hs.is_mine(domain_specific_string)
+
+    def is_mine_id(self, string: str) -> bool:
+        """Checks whether an ID comes from this homeserver."""
+        return string.split(":", 1)[1] == self._server_name
```

Comment:
Could you please rename this into `is_mine_str` or similar and extend the docstring to explain how it differs from `is_mine`? As it stands it feels a bit confusing to the external module developer.

Reply:
Or extend `is_mine` to allow passing user IDs are `str`s as well?

Reply:
Ah yeah that'd probably be a better way of doing it.

Reply:
TBF `is_mine_id` is consistent with the function in `synapse.types`, and the convention where e.g. `user` is an object and `user_id` is a string. :shrug:

Reply:
Yeah tbh I find how it's done in `synapse.types` confusing, so I think it makes sense to make it not confusing when the target audience isn't Synapse devs.